content/en/continuous_integration/guides/add_custom_metrics.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Add Custom Metrics to your Tests
+kind: guide
+---
+
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+This guide will walk you through adding and using custom metrics for your tests.
+
+### Add the custom metric to your test
+First step is to add the custom metric to your test by using the programmatic API:
+
+{{< tabs >}}
+{{% tab "Javascript/Typescript" %}}
+```javascript
+  it('sum function can sum', () => {
+    const testSpan = require('dd-trace').scope().active()
+    testSpan.setTag('test.memory.rss', process.memoryUsage().rss)
+    // test continues normally
+    // ...
+  })
+```
+{{% /tab %}}
+
+{{% tab ".NET" %}}
+```csharp
+// inside your test
+var scope = Tracer.Instance.ActiveScope; // from Datadog.Trace;
+if (scope != null) {
+    scope.Span.SetTag("test.memory.usage", 1e8);
+}
+// test continues normally
+// ...
+```
+{{% /tab %}}
+
+{{% tab "Java" %}}
+```java
+// inside your test
+final Span span = GlobalTracer.get().activeSpan();
+if (span != null) {
+  span.setTag("test.memory.usage", 1e8);
```

Comment:
FIX: I don't know if `setTag` allows numerical values

Reply:
Yes, there is a `Span setTag(String var1, Number var2)` method.

Reply:
nice, thanks